src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -53,7 +53,8 @@ export default ajaxCaller(class LaunchAnalysisModal extends Component {
     }, [
       !message && !launchError && div('Confirm launch'),
       message && div([spinner({ style: { marginRight: '0.5rem' } }), message]),
-      launchError && div({ style: { color: colors.danger() } }, [launchError])
+      launchError && div({ style: { color: colors.danger() } }, [launchError]),
+      h(CromwellVersionLink, { style: { display: 'block', marginTop: '1rem' } })
```

Comment:
This looks a little out of place here, without any context. Thoughts?

Reply:
Tweaked with Sushma!